src/cartridges/int_adyen_SFRA/cartridge/templates/default/checkout/billing/adyenComponentForm.isml

```diff
@@ -21,6 +21,7 @@
       window.paymentFromComponentURL = "${URLUtils.https('Adyen-PaymentFromComponent')}";
       window.resultCodeAuthorised = '${AdyenConstants.RESULTCODES.AUTHORISED}';
       window.paymentsDetailsURL = "${URLUtils.https('Adyen-PaymentsDetails')}";
+      window.ShowConfirmationPaymentFromComponent = "${ShowConfirmationPaymentFromComponent}";
```

Comment:
Why do we need this as `window.ShowConfirmationPaymentFromComponent`? Is it used somewhere else?

Reply:
now after the review changes yes :P
Before that I was calling `"${ShowConfirmationPaymentFromComponent}"` directly but it still had to be defined in the template, otherwise it's undefined